hyperhtml.js

```diff
@@ -1,56 +1,43 @@
-var hyperHTML = (function () {'use strict';
+var hyperHTML = (function (document) {'use strict';
```

Comment:
the only reason it's there is for minifiers but it makes sense to give it a better name.